### DISCUSSION TITLE
failed to deploy!!!

### DISCUSSION THREAD

**Discussion by sably** [vercel/next.js#49814] [Help] [answered]
### Summary

it's my first deploy at Vercel and I got an error.
take a look: 
![Screenshot 2023-05-15 185340](https://github.com/vercel/next.js/assets/25221341/0e73219c-71c3-4b34-8794-6011e03dda09)

**Comment by sably** [comment]
how can i fix that?

**Comment by devjiwonchoi** [comment]
Double check your import, the import should be for example,
if it is exported as export default, then it could be such as
`import Layout from "../../components/layout"`
or else, it should be
`import { Layout } from "../../components/layout/Layout"`

**Reply by sably** [reply]
i just see one import...

**Reply by devjiwonchoi** [reply]
could you show your code?

**Reply by sably** [reply]
https://github.com/sably/nextjs-blog

**Reply by devjiwonchoi** [reply]
I've cloned your repo and build successed, but seems like a different issue. Your layout.js has two props, home and children, and "home" props is not being passed from [id].js. Make sure you pass home data as follows
![Screenshot 2023-05-15 at 11 41 44 PM](https://github.com/vercel/next.js/assets/120007119/e8543622-e8c6-4640-8437-578754110947)

**Reply by sably** [reply]
ty for ur response icyJoseph solved that

**Comment by icyJoseph** [comment, marked as answer]
Hi,

You've been bitten by case-sensitivity.

You import like this:

```js
import Layout from "../components/Layout";
```

But the file `components/Layout` (with capital L) doesn't exist! Instead you have `components/layout.js` (included the extension for completeness)

You have to use:

```js
import Layout from "../components/layout";
```

Notice the lowercased `l`.

Linux is case sensitive, so `components/layout.js`, `components/Layout.js`, are effectively, two different files! Not the same in MacOS  and Windows IIRC. 

From https://learn.microsoft.com/en-us/windows/wsl/case-sensitivity:

> Standard behavior:
> 
> Windows file system treats file and directory names as case-insensitive. FOO.txt and foo.txt will be treated as equivalent files.
> Linux file system treats file and directory names as case-sensitive. FOO.txt and foo.txt will be treated as distinct files.
>
